hack/make-rules/helpers/cache_go_dirs.sh

```diff
@@ -60,9 +60,9 @@ fi
 mkdir -p $(dirname "${CACHE}")
 if $("${NEED_FIND}"); then
     kfind -type f -name \*.go  \
-        | xargs -n1 dirname    \
-        | LC_ALL=C sort -u     \
+        | sed 's|/[^/]*$||'    \
```

Comment:
I don't expect much improvement. The issue was that dirname was called once per file. Sed only runs once.